tests/webp-uploads-test.php

```diff
@@ -0,0 +1,39 @@
+<?php
+/**
+ * Tests for webp-uploads module.
+ *
+ * @package performance-lab
+ */
```

Comment:
Can you put this file into a `tests/modules/webp-uploads` directory? That's how it's specified in the documentation for writing a module, module tests should be in their own directory like that.

Reply:
will do.  makes sense, missed that bit in the docs.